.github/workflows/cdn_tests.yml

```diff
@@ -55,6 +55,8 @@ jobs:
       PYTEST_PROCESSES: auto
       SAUCELABS_API_KEY: ""
       SAUCELABS_USERNAME: ""
+      RERUNS_ALLOWED: 3
+      RERUNS_DELAY_SECS: 60
```

Comment:
This 60 is a guess for now. We might be able to dial it up to longer, without the GHA timing out, or we might not. Will have to see